pkg/extensions/tbtc/tbtc.go

```diff
@@ -417,9 +515,6 @@ type backoffFn func(iteration int) time.Duration
 
 type timeoutFn func(depositAddress string) (time.Duration, error)
 
-// TODO:
-//  1. Handle chain reorgs (keep-ecdsa/pull/585#discussion_r511760283 and keep-ecdsa/pull/585#discussion_r513447505)
-//  2. Resume monitoring after client restart.
```

Comment:
Point 2 should stay, correct?

Reply:
I removed this because I think we should first discuss if we really need restart handling and if so, how we should approach it. Because of that, I think we should rather create a separate GH issue than leave a TODO.

Reply:
Oh, there will be an issue today, that's for sure :) This TODO was just like a comment for someone else browsing this code and wondering how restarts are handled.

Reply:
Sure I can leave that TODO then.